common/scala/src/main/resources/logging.conf

```diff
@@ -1,4 +1,5 @@
 akka {
   loglevel = "INFO"
```

Comment:
You can now safely set this to `DEBUG`, so the loglevel is completely controlled by the backend and not akka itself.

Reply:
If I understand Akka logging right, setting the level to DEBUG will cause that all messages will be placed into the event stream and only afterward filtered out by Logback. Therefore I would prefer to adapt the log level after we have reduced the number of messages with subsequent changes.

Reply:
I tried it out. If you use the `LoggingFilter` while still setting "DEBUG" in akka, `isEnabled(DebugLevel)` on the logger returns `false` (it seems to take the value from the configured backend then).

And per documentation:

> You need to enable the Slf4jLogger in the loggers element in the configuration. Here you can also define the log level of the event bus. More fine grained log levels can be defined in the configuration of the SLF4J backend (e.g. logback.xml). You should also define akka.event.slf4j.Slf4jLoggingFilter in the logging-filter configuration property. It will filter the log events using the backend configuration (e.g. logback.xml) before they are published to the event bus.

Reply:
Changed as amend to the existing commit